architecture/revhistory_architecture.adoc

```diff
@@ -4,6 +4,31 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+
+// tag::architecture_wed_apr_27_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+|link:../architecture/core_concepts/routes.html[Core Concepts -> Routes]
+|Corrected footnote placement in the link:../architecture/core_concepts/routes.html#secured-routes[Secured Routes] section.
+
+n|link:../architecture/core_concepts/builds_and_image_streams.html[Core Concepts -> Builds and Image Streams]
+|Added information about maximum name lengths for objects.
+
+n|link:../architecture/core_concepts/projects_and_users.html[Core Concepts -> Projects and Users]
```

Comment:
@adellape Thanks.  Yes, this is related to the recent changes.  I have submitted a merge request (no.9) w/ a fix for the spurious 'n', and one (no.10) for the "out of place" intro sentence.